sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/image/image_classification_job.py

```diff
@@ -37,6 +37,15 @@ def __init__(
 
         :param primary_metric: The primary metric to use for optimization
         :param kwargs: Job-specific arguments
+
```

Comment:
General comment: 
[doc guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/documentation_guidelines.md#clarity) indicate that we should have docstrings for class definition not for `__init__()`. Consider moving the docstrings to `__init__()` to class definition

Reply:
Thanks for mentioning that.